deploy-env/conda/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -
 
 RUN source ~/miniconda3/bin/activate && \
     conda config --add channels conda-forge && \
-    conda install -c lsstts python=3.7 ts-salobj="v5.3.0" cython ts-idl="1.1.0_v4.6.0"
+    conda install -c lsstts python=3.7 ts-salobj="v5.4.1_RC1" cython ts-idl="v1.1.1_v4.6.0"
```

Comment:
We need to watch using RCs for deployment environments.

Reply:
This will be superseded by Rob's work on the deployment container build.

Reply:
Great.